src/helics/application_api/FederateInfo.cpp

```diff
@@ -37,37 +38,64 @@ FederateInfo::FederateInfo(const std::string& args)
     loadInfoFromArgsIgnoreOutput(args);
 }
 
-static const std::map<std::string, int> propStringsTranslations{
+static const std::unordered_map<std::string, int> propStringsTranslations{
     {"period", helics_property_time_period},
+    {"timeperiod", helics_property_time_period},
+    {"timeperiod", helics_property_time_period},
```

Comment:
Aren't these two lines the same?

```suggestion
    {"timeperiod", helics_property_time_period},
```

Reply:
I think I meant to add an underscore